src/cogent3/core/moltype.py

```diff
@@ -742,6 +742,17 @@ def to_json(self):
         data = self.to_rich_dict(for_pickle=False)
         return json.dumps(data)
 
+    def to_regex(self, seq):
+        """returns regular expression for input sequence"""
```

Comment:
change docstring to "returns a regex pattern with ambiguities expanded to a character set"